Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Amazon.Lambda.Annotations.SourceGenerator.csproj

```diff
@@ -1,26 +1,50 @@
+    <PackageIcon>images\icon.png</PackageIcon>
+    <PackageReadmeFile>docs\README.md</PackageReadmeFile>
+    
+    <!--This assembly needs to access internal methods inside the Amazon.Lambda.Annotations.Models assembly.
     Both these assemblies need to be strongly signed for the InternalsVisibleTo attribute to take effect.-->
     <AssemblyOriginatorKeyFile>..\..\..\buildtools\public.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly> 
 
     <!-- This is required to allow copying all the dependencies to bin directory which can be copied after to nuget package based on nuspec -->
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    
+    <Version>1.5.3</Version>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis" Version="4.0.1" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.0.1" />
-    <PackageReference Include="System.CodeDom" Version="6.0.0" />
+    <None Include="$(OutputPath)\netstandard2.0\Amazon.Lambda.Annotations.Models.dll" Pack="true" PackagePath="lib/netstandard2.0" Visible="false" />
+    <None Include="$(OutputPath)\netstandard2.0\Amazon.Lambda.Annotations.Models.xml" Pack="true" PackagePath="lib/netstandard2.0" Visible="false" />
+    <None Include="$(OutputPath)\net6.0\Amazon.Lambda.Annotations.Models.dll" Pack="true" PackagePath="lib/net6.0" Visible="false" />
+    <None Include="$(OutputPath)\net6.0\Amazon.Lambda.Annotations.Models.xml" Pack="true" PackagePath="lib/net6.0" Visible="false" />
+    <None Include="$(OutputPath)\net8.0\Amazon.Lambda.Annotations.Models.dll" Pack="true" PackagePath="lib/net8.0" Visible="false" />
+    <None Include="$(OutputPath)\net8.0\Amazon.Lambda.Annotations.Models.xml" Pack="true" PackagePath="lib/net8.0" Visible="false" />
+    
+    <None Include="$(OutputPath)\netstandard2.0\$(AssemblyName).dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
+    <None Include="$(OutputPath)\netstandard2.0\Amazon.Lambda.Annotations.Models.dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
+    <None Include="$(OutputPath)\netstandard2.0\Newtonsoft.Json.dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
+    <None Include="$(OutputPath)\netstandard2.0\YamlDotNet.dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis" Version="4.0.1" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.0.1" PrivateAssets="all" />
+    <PackageReference Include="System.CodeDom" Version="6.0.0" PrivateAssets="all" />
   </ItemGroup>
 
   <ItemGroup>
-    <!-- Generator dependencies, update Libraries/src/Amazon.Lambda.Annotations.nuspec whenever a new generator dependency is added.  -->
+    <PackageReference Include="Amazon.Lambda.Core" Version="2.1.0" />
```

Comment:
Convert this to a `ProjectReference`. It was only a pinned version in the nuspec because you can't use `ProjectReferences` in the nuspec file.

Reply:
Done